src/components/MapBox.tsx

```diff
@@ -127,6 +150,9 @@ export const MapBox: React.FC = () => {
   const { selectedFacilityId } = useAppState();
   const dispatch = useAppDispatch();
 
+  const theme = useTheme();
+  const isMobileView = useMediaQuery(theme.breakpoints.down('md'));
```

Comment:
in the `SearchForm` it is refered to as `isMobile` and here it is called `isMobileView`. In `SearchCard` it is called `medium`. They all use the same breakpoints. 

Let's pick one name to keep it consistent.

Reply:
arrived from main